lib/cmds/files_cmds/mv.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const chalk = require('chalk');
+const querystring = require('querystring');
+const path = require('path');
+const { patch, list } = require('../../api');
+
+exports.command = 'mv <source> <dest>';
+exports.desc = 'Move/rename files from <source> to <dest> within a project';
+exports.builder = yargs => {
+  yargs.positional('source', {
+    describe: 'Either the ID of the file to move or a project ID and prefix of the form <project ID>/<prefix> to move a set of files in a project.',
+    type: 'string'
+  }).positional('dest', {
+    describe: 'The destination directory for the download file.  Defaults to current working directory',
```

Comment:
This descriptions seems wrong, because it talks about downloading files rather than moving them.